index.js

```diff
@@ -22,6 +22,10 @@ mongoose.connect(
   },
 );
 
+app.use(express.static('public'));
+
+app.get('/', (req, res) => res.redirect('/views/index.html'));
```

Comment:
I can only redirect files inside the folder declared in `express.static()`?

Reply:
It's redundant! I already updated it.